utils/node_local_test.py

```diff
@@ -3988,6 +3988,76 @@ def test_pydaos_kv(server, conf):
     daos._cleanup()
     log_test(conf, pydaos_log_file.name)
 
+
+def test_pydaos_kv_obj_class(server, conf):
+    """Test the predefined object class works with KV"""
+    # pylint: disable=consider-using-with
+
+    pydaos_log_file = tempfile.NamedTemporaryFile(prefix='kv_objclass_pydaos_',
+                                                  suffix='.log',
+                                                  delete=False)
+
+    os.environ['D_LOG_FILE'] = pydaos_log_file.name
```

Comment:
Pre-existing in other places in this file, so out of scope - question for @ashleypittman If we just need the name, and `delete=False`, then shouldn't this work?
```suggestion
    with tempfile.NamedTemporaryFile(prefix='kv_objclass_pydaos_',
                                     suffix='.log',
                                     delete=False) as tmp_file:
        os.environ['D_LOG_FILE'] = tmp_file.name
```

Reply:
Yes, that would be preferable, and then the "disable=consider-using-with" pragma can be removed as well.

It's one fewer local, one fewer suppression and the code is clearer in it's intent.